### PR TITLE
Add Service Fabric artifact.

### PR DESCRIPTION
**Pull request by togores-msft**
Add Service Fabric artifact which does the following:
 - Installs Chocolatey.
 - Installs WebPICmd (Web Platform Installer Command Line) via Chocolatey.
 - Installs latest available version of Microsoft Azure Service Fabric SDK for Visual Studio 2015 and/or 15.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@togores-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tiandra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: Artifacts/windows-servicefabric/StartChocolatey.ps1 [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+$exitCodeVS2015 = Try-Install-Service-Fabric "14.0" "VS2015" $credential
+$exitCodeDev15 = Try-Install-Service-Fabric "15.0" "Dev15" $credential
+
+Disable-PSRemoting -Force
+
+# Delete the artifactInstaller user
+$cn.Delete("User", $userName)
+
+# Delete the artifactInstaller user profile
+gwmi win32_userprofile | where { $_.LocalPath -like "*$userName*" } | foreach { $_.Delete() }
```
Comment (leovms) [inline Artifacts/windows-servicefabric/StartChocolatey.ps1]: Consider putting the main logic in a try / finally block to ensure calls to Disable-PSRemoting and $cn.Delete happen.

File: Artifacts/windows-servicefabric/Artifactfile.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "Service Fabric",
```
Comment (leovms) [inline Artifacts/windows-servicefabric/Artifactfile.json]: Service Fabric SDK?

**Comment by leovms** [comment]
Ran artifact on Windows Server 2008 R2 SP1. Ran into some errors. Please review and correct as needed.

[Service Fabric Artifact Logs.zip](https://github.com/Azure/azure-devtestlab/files/303235/Service.Fabric.Artifact.Logs.zip)

**Comment by leovms** [comment]
Thanks for the updates and the contribution. Merging your changes.

**Pull request merged**